src/parser.ts

```diff
@@ -60,7 +61,6 @@ export class Parser {
     readonly scanner: Scanner;
     readonly operatorPrecedence: any;
 
-    readonly sourceType: string;
```

Comment:
That's right. This variable is internal to the parser since the only use of `sourceType` was to compare it against `module`.

`sourceType` as a visible property of `Program` node still exists and hasn't changed.